src/module_loader.rs

```diff
@@ -3,7 +3,7 @@ use crate::{
     transpiler,
 };
 use deno_core::{
-    anyhow::{self, anyhow},
+    anyhow::{self, anyhow, Ok},

```

Comment:
What is anyhow::Ok for here

Reply:
I may have accidentally tab-completed that into existence from somewhere else in the file. Will try regular Ok later today.

Reply:
Yeah, looks fine without it.